kytos/utils/napps.py

```diff
@@ -429,6 +443,13 @@ def create_ui_structure(cls, username, napp_name, ui_templates_path,
                                               context)
                 file.write(content)
 
+    @classmethod
+    def create_requirements_structure(cls, username, napp_name):
+        """Create the requirements directory structure."""
+        os.makedirs(os.path.join(username, napp_name, 'requirements'))
+        for file in ['dev.txt', 'dev.in', 'run.in']:
+            open(os.path.join(username, napp_name, 'requirements', file), 'w')
```

Comment:
It's not a good practice to open files without closing them - they may end up being not written to the disk at all.